docs/contactus.rst

```diff
@@ -15,24 +15,15 @@ for SUMO community discussions.
 Kitsune hackers
 ===============
 
-If you're hacking on the Kitsune code and have questions, ping us on
-IRC.
+If you're hacking on the Kitsune code and have questions, ping us on `Matrix <https://wiki.mozilla.org/Matrix>`_.
 
-We hang out in ``#sumodev`` on irc.mozilla.org. We're usually around
-during the day in Eastern Time.
+We hang out in `#sumodev:mozilla.org <https://chat.mozilla.org/#/room/#sumodev:mozilla.org>`_.
```

Comment:
Similar let's change this to support-platform. I would remove any reference to timezone. We should have 24h coverage to at least acknowledge a request/question.